packages/orbit-components/src/ListChoice/index.tsx

```diff
@@ -108,32 +32,52 @@ const ListChoice = React.forwardRef<HTMLDivElement, Props>(
     };
 
     return (
-      <StyledListChoice
+      <div
+        className={cx(
+          "orbit-list-choice",
+          "py-sm px-md box-border flex w-full items-center",
+          "border-b-cloud-normal bg-white-normal border-b-solid border-b",
+          "duration-fast transition-[background-color] ease-in-out",
+          disabled ? "cursor-not-allowed" : "cursor-pointer",
+          "hover:outline-none [&:hover_button]:bg-transparent",
```

Comment:
TIL that `[&:hover_button]:` is the same as `[&_button]:hover:`. We have been using the last one in every other components.

I honestly have no preference. But for the sake of consistency I'd advise for a change here.

I even tried if this would work with the storybook pseudo-states addon, but it also doesn't 😞

Reply:
Whoops, actually took me a while to come up with that. Don't why I thought `hover:` must be always at the beginning of the class.